src/modules/content-tree/services/content.tree.service.js

```diff
@@ -14,17 +14,17 @@ export const loadLocationItems = (parentLocationId, callback, limit = 50, offset
         },
     });
 
-    fetch(request)
-        .then(handleRequestResponse)
-        .then((data) => {
-            const location = data.ContentTreeNode;
+    // fetch(request)
+    //     .then(handleRequestResponse)
+    //     .then((data) => {
+    //         const location = data.ContentTreeNode;
 
-            location.children = location.children.map(mapChildrenToSubitems)
+    //         location.children = location.children.map(mapChildrenToSubitems)
 
-            return mapChildrenToSubitems(location);
-        })
-        .then(callback)
-        .catch(showErrorNotification);
+    //         return mapChildrenToSubitems(location);
+    //     })
+    //     .then(callback)
+    //     .catch(showErrorNotification);
```

Comment:
Why is this catch not working? 
The result of this fetch when there is no endpoint should be a notification at the bottom.

Committing commented out code is never a good solution.